src/compiler/checker.ts

```diff
@@ -11362,30 +11589,62 @@ namespace ts {
         }
 
         /**
-         * Given an opening/self-closing element, get the 'element attributes type', i.e. the type that tells
-         * us which attributes are valid on a given element.
+         * Get attributes type of the given intrinsic opening-like Jsx element by resolving the tag name.
+         * The function is intended to be called from a function which has checked that the opening element is an intrinsic element.
+         * @param node an intrinsic JSX oopening-like element

```

Comment:
1. typo:opening
2. can you add an assert to verify that it's an intrinsic attribute, like `Debug.assert(symbol)` after calling getIntrinsicTagSymbol or something?